debian/control

```diff
@@ -9,7 +9,7 @@ Build-Depends: debhelper-compat (= 13),
                pkg-config,
                zlib1g-dev,
                libyajl-dev (>= 2)
-Standards-Version: 4.5.0
+Standards-Version: 4.6.0
 Homepage: https://github.com/edenhill/kafkacat
```

Comment:
The project has been renamed to kcat, the URLs should be updated accordingly.

Reply:
The bot now follows renames, and has updated these URLs.